nexus/pkg/monitor/info.go

```diff
@@ -0,0 +1,135 @@
+package monitor
```

Comment:
todo:
- move this logic back to the handler. There is no special need for this file.
- what to do with requirements/conda?
- need to figure out what to do with git.